linuxdoc/manKernelDoc.py

```diff
@@ -55,7 +55,6 @@ def setup(app):
 
     app.add_builder(KernelDocManBuilder)
     app.add_directive("kernel-doc-man", KernelDocMan)
-    app.add_config_value('author', "", 'env')
     app.add_node(kernel_doc_man
```

Comment:
@GabrielGanne  thanks for heads up! .. merged your patch, see  b2a90c7

FYI: I add SPHINXVERS variable to select sphinx-doc version for the build / with we can test with different sphinx-doc versions, see commit 6d2bbbc181

About the rest of your PR: hope to find the time to continue the review tomorrow / anyway, first let me say "thanks" for your efforts.

Reply:
Hi,

So I did test and have a slight issue with this: your commit which just follows the introduction of SPHINXVERS (88f1b8e43ad5dca635e0be95133c4f1220353dcc) makes use of the new sphinx logging API (introduced about two years ago in commit https://github.com/sphinx-doc/sphinx/commit/6d4e6454093953943e79d4db6efeb17390870e62, which means tag >= 1.6).

However, I'm working with debian stable which is a tad behind with sphinx 1.4.9-2.
Even though I _could_ install sphinx from pip instead, I'd prefer to keep using the distrib's packages if possible.

I'll see if there's a way to make it work with all versions.

Reply:
Aaargh, the downward compatibility of Sphinx is horrible. Give me a moment, I will take a look.

Reply:
@GabrielGanne I added a compat.py to address this issue https://github.com/return42/linuxdoc/commit/0521ac817a588bd13c342814084fde969c900989. I hope this helps you for now. 

Please read the commit message; IMO we should not investigate to much time in downward compatibility. Instead it might be better to serve a tarball or a (snap) package for a complete install on a restricted (offline) build host.